packages/pn-commons/src/components/__test__/CustomDatePicker.test.tsx

```diff
@@ -51,11 +60,15 @@ describe('test CustomDatePicker languages', () => {
   ];
 
   it.each(languages)('check january month to be $month in $language', (language) => {
-    const { container, getByRole } = render(<RenderDatePicker language={language.language} />);
+    const { container, getByLabelText } = render(<RenderDatePicker language={language.language} />);
     expect(container).toHaveTextContent(/datepicker/i);
-    const button = getByRole('button');
+    const button = getByLabelText(/DatePicker/i);
     fireEvent.click(button);
-    const regExMonth = new RegExp(`${language.month}`, 'i');
-    expect(screen.getByText(regExMonth)).toBeInTheDocument();
+    const regExMonth = new RegExp(`${language.month}`);
```

Comment:
why this change?

Reply:
same as before commnet

Reply:
can you copy the comment here? i don't find the comment

Reply:
same as before